alert_on_new_query_results.py

```diff
@@ -68,7 +69,7 @@ def saved_search_to_query(saved_search: Dict) -> Dict:
     return saved_search
 
 
-def filter_listings(query_json: Dict, listings: List[Dict]) -> List[Dict]:
+def filter_listings(query_json: Dict, listings: List[Dict], query_name: str, filters: Dict) -> List[Dict]:
```

Comment:
Please add docstring comments for query_name and filters

If possible, specify the type of filters past just `Dict`

Reply:
While I added a docstring, I was confused by what you meant on types of filters past Dict.

Reply:
Ahh, I haven't used the typing library before so it wasn't obvious to me, well I suppose in the future the query_name global_filters or something similar could be reserved just for global filters instead of using a higher level, then the type would be Dict[str, Dict[str,str]], at least so far.  I'd have to check if typing.Union is needed support the case of an empty Dict as well.

I'll keep that in consideration if I end up making more filters that might be helpful.